README.md

```diff
@@ -1,2 +1,61 @@
+
+### Performance
+
+Well thought out efforts, and focuses mainly on fast paced performance to render 100k data in less than a second.
+
+### Trendlines
+
+Shows the price direction and movement speed. Trendlines can be generated for Cartesian type series (Line, Column, Scatter, Area, Candle, HiLo, etc.) except bar. More than one trendline can be added to a series.
+
+### Exporting
+Export Charts to PDF documents or as image formats such as SVG, PNG and JPEG, as well as export chart data to XLSX and CSV formats.
+
+## Related links
+[Learn More about Vue Charts](https://www.syncfusion.com/vue-components/vue-charts?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/account/manage-trials/downloads?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples)
+
+[Documentation](https://ej2.syncfusion.com/vue/documentation/chart/getting-started?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-vue-charts-component?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/vue-components?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/vue?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples)
+
+[Online example](https://ej2.syncfusion.com/vue/demos/#/bootstrap5/chart/over-view.html?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples)
+
+
+
+## About Syncfusion Vue Components
+
+Syncfusion's [Vue Components](https://www.syncfusion.com/vue-components?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples) library is the only suite that you will ever need to build an application since it contains over 80 high-performance, lightweight, modular, and responsive UI Components in a single package. In addition to Charts, we provide popular Vue Components such as [Button](https://www.syncfusion.com/vue-components/vue-button?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples), [Badge](https://www.syncfusion.com/vue-components/vue-badge?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples), [Scheduler](https://www.syncfusion.com/vue-components/vue-scheduler?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples), [Diagram](https://www.syncfusion.com/vue-components/vue-diagram?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples), and [Word Processor](https://www.syncfusion.com/vue-components/vue-word-processor?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples).
+
```

Comment:
Please check on this, since u are working on charts no need to add here.
![image](https://github.com/SyncfusionExamples/getting-started-with-the-vue-charts-component/assets/95272441/9fd19c39-8e98-4996-a78c-4a2367fbfbe1)

Reply:
Corrected